Added build & run instructions to the readme

Laoujin opened the pull request.

Laoujin committed:
Added build & run instructions to the readme

spion commented:
Docpad should automatically start a server when you run npm start, shouldn't it?

Laoujin committed:
Readme: added docpad info

Laoujin commented:
hehehe oops... that's what happens when you do not read the "info: " lines :)
I updated the pull request...

There are also two READMEs... I deleted one of them?

spion commented:
thanks!
